amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/polling/PollingConfig.java

```diff
@@ -119,14 +125,23 @@ public PollingConfig(String streamName, KinesisAsyncClient kinesisClient) {
      */
     private RecordsFetcherFactory recordsFetcherFactory = new SimpleRecordsFetcherFactory();
 
+    /**
+     * @Deprecated Use {@link PollingConfig#idleTimeBetweenReadsInMillis} instead
+     */
+    @Deprecated
+    public void setIdleTimeBetweenReadsInMillis(long idleTimeBetweenReadsInMillis) {
```

Comment:
What is going to be the cost of deprecating this. I know that we like everything to be backward compadible. Do we know how many people use this. What will happen to peoples code as they update. Will this require a major release?

It looks to me that it will just call the value below

Reply:
Deprecating will print a warning during compilation for customer still using this. The behavior will still be correct because they both do the same thing.

We want to avoid confusion with multiple ways to do the same thing, so this says explicitly that we want to support the lombok-fluent-like API that customers are already used to using over the other setter.